nuget.config

```diff
@@ -4,6 +4,7 @@
     <clear />
```

Comment:
@TravisEz13, your last commit had 3 failures in `PowerShell-CI-linux`
DSC MOF Compilation.Should be able to compile a MOF from a basic configuration
```powershell
Expected no exception to be thrown, but an exception "Errors occurred while processing configuration 'DSCTestConfig'." was thrown from /home/vsts/work/1/a/bins/publish/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.psm1:3710 char:5
    +     throw $ErrorRecord
    +     ~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 30
30: "@) | Should -Not -Throw
```
DSC MOF Compilation.Should be able to compile a MOF from another basic configuration
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 nxScript f1
+                 ~~~~~~~~
Undefined DSC resource 'nxScript'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 36
36:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from a complex configuration
```powershell
Expected no exception to be thrown, but an exception "Errors occurred while processing configuration 'WordPressServer'." was thrown from /home/vsts/work/1/a/bins/publish/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.psm1:3710 char:5
    +     throw $ErrorRecord
    +     ~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 161
161: "@) | Should -Not -Throw
```